client/my-sites/checkout/checkout/source-payment-box.jsx

```diff
@@ -151,6 +175,17 @@ class SourcePaymentBox extends PureComponent {
 						onChange={ this.handleChange }
 						label={ translate( 'Your Name' ) }
 						eventFormName="Checkout Form" />
+					<div className="checkout-field">
+						<FormLabel htmlFor="ideal-bank">
+							{ translate( 'Bank' ) }
+						</FormLabel>
+						<FormSelect
+							name="ideal-bank"
+							onChange={ this.handleChange }
```

Comment:
Shouldn't we have `handleChange` verify that not `-` was chosen? While I understand it should work without bank, it's still a bit pointless to allow the form to be sumitted with `Select bank...` in the dropdown selected. I understand being validated on the server-side at least for not being empty, so probably this is a _maybe_.

Reply:
We should add client-side validation at some point, but for now I think it's ok to rely on the server side (like we do with Paypal). W